x/kavamint/keeper/grpc_query.go

```diff
@@ -24,3 +29,57 @@ func (k Keeper) Inflation(c context.Context, _ *types.QueryInflationRequest) (*t
 
 	return &types.QueryInflationResponse{Inflation: inflation}, nil
 }
+
+// MintQueryServer implements cosmos sdk's x/mint querier.
+// x/mint was removed from kava, but the standard inflation endpoint is still registered
+// for easier third party integration and backwards compatibility.
+type MintQueryServer struct {
+	keeper Keeper
+}
+
+// NewMintQueryServer returns a service that implements x/mint's QueryServer
+func NewMintQueryServer(kavamintKeeper Keeper) MintQueryServer {
+	return MintQueryServer{kavamintKeeper}
+}
+
+var _ minttypes.QueryServer = MintQueryServer{}
+
+// Params is not implemented. There is no mint module.
+func (MintQueryServer) Params(
+	_ context.Context, _ *minttypes.QueryParamsRequest,
+) (*minttypes.QueryParamsResponse, error) {
+	return nil, status.Error(codes.Unimplemented, "x/mint has been replaced by x/kavamint")
```

Comment:
Like the additional info pointing callers to the kavamint info.  This could also be extracted to a method since it's duplicated, but OK to leave since it's only used twice.  Likely not worth the extra layer of abstraction